### DISCUSSION TITLE
How we can add or create game in Resource?

### DISCUSSION THREAD

**Discussion by sachinb80** [Sunbird-Knowlg/Community#115] [Q&A]
I would need to understand how this part works.
How to add a game in Resource?
![image](https://user-images.githubusercontent.com/109504832/227850052-2e4b94b1-15ce-47a2-aff3-0a3f8855f992.png)
![image](https://user-images.githubusercontent.com/109504832/227850266-9c8098af-79cc-47f6-86d9-dbce462867f9.png)

**Comment by pallakartheekreddy** [comment]
@sachinb80 Creating a resource we will use a content editor and content editor built using a plugin framework. As shown above, screenshot `activity browser` browser shows a list of different category plugins which we can add it to any slide inside of resource. Please refer below document for more details
https://knowlg.sunbird.org/learn/product-and-developer-guide/editors/how-to-contribute-as-a-js-plugin/interactive-editor-plugin-guide
https://knowlg.sunbird.org/learn/product-and-developer-guide/editors/how-to-contribute-as-a-js-plugin/how-do-i-think-about-a-plugin

**Reply by pallakartheekreddy** [reply]
@sachinb80 Pls let us know if your query is resolved.

**Reply by pallakartheekreddy** [reply]
@sachinb80 As we haven't received a response from you, we will proceed to close this thread. If you have any further questions or require assistance in the future, please don't hesitate to start a new thread. Thank you.
